client/src/components/categorical/value.js

```diff
@@ -26,12 +30,16 @@ import { AnnotationsHelpers } from "../../util/stateManager";
   colorAccessor: state.colors.colorAccessor,
   schema: state.world?.schema,
   world: state.world,
-  crossfilter: state.crossfilter
+  crossfilter: state.crossfilter,
+  ontology: state.ontology,
+  ontologyLoading: state.ontology?.loading,
+  ontologyEnabled: state.ontology?.enabled
 }))
 class CategoryValue extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      activeSuggestItem: "",
```

Comment:
this state value seems unused?

Reply:
Yes and no... AnnoInputs passes a function here:

handleItemChange={this.handleSuggestActiveItemChange}

and we store the value, but do not currently use it. Conceivable that we would. Remove?

Reply:
same as above: keep it clean....